homeassistant/components/apple_tv/remote.py

```diff
@@ -38,16 +38,17 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Load Apple TV remote based on a config entry."""
-    name = config_entry.data[CONF_NAME]
-    manager = hass.data[DOMAIN][config_entry.unique_id]
-    async_add_entities([AppleTVRemote(name, config_entry.unique_id, manager)])
+    name: str = config_entry.data[CONF_NAME]
+    if config_entry.unique_id is not None:
+        manager: AppleTVManager = hass.data[DOMAIN][config_entry.unique_id]
+        async_add_entities([AppleTVRemote(name, config_entry.unique_id, manager)])
```

Comment:
Is this valid, i.e. it's possible that the config entry does not have a `unique_id`?
If it's not possible, this should be changed to:
```suggestion
    # apple_tv config entries always have a unique id
    assert config_entry.unique_id is not None
    manager: AppleTVManager = hass.data[DOMAIN][config_entry.unique_id]
    async_add_entities([AppleTVRemote(name, config_entry.unique_id, manager)])
```